src/integrations/prefect-dbt/tests/cli/test_commands.py

```diff
@@ -128,23 +185,25 @@ def test_trigger_dbt_cli_command_find_home(dbt_cli_profile_bare):
     @flow
     def test_flow():
         return trigger_dbt_cli_command(
-            "dbt ls", dbt_cli_profile=dbt_cli_profile, overwrite_profiles=False
+            "ls", dbt_cli_profile=dbt_cli_profile, overwrite_profiles=False
```

Comment:
For backwards compatibility, can we support passing in `dbt` as part of the command? We should be able to strip a leading `dbt` from the `command` before passing it to the `dbt_runner`.